src/index.js

```diff
@@ -132,19 +131,46 @@ function addTask() {
 // remove task
 
 // edit task
+function markTaskAsUncompleted(taskElement) {
+  taskElement.classList.remove('completed');
+  const checkbox = taskElement.querySelector('input[type="checkbox"]');
+  checkbox.checked = false;
+}
```

Comment:
- [ ] Please let's try to make todo more interactive. As attached in the video, and as [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md) kindly ensure that when the checkbox is ticked when the complete status changes to `true` and `false` when unchecked. The changes should be saved in the localstorage. Kindly refer to the video also for clarity.